db/models.py

```diff
@@ -50,3 +55,65 @@
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
```

Comment:
The `user` field in the `Order` model uses `settings.AUTH_USER_MODEL`, which requires `from django.conf import settings` to be imported. Ensure that this import is correctly added at the top of the file.